autosklearn/automl.py

```diff
@@ -1011,7 +1012,13 @@ def fit_pipeline(
 
         # Get the components to include and exclude on the configuration space
         # from the estimator attributes
-        include, exclude = self._get_include_exclude_pipeline_dicts()
+        include, exclude = parse_include_exclude_components(
+            task=self._task,
+            include_estimators=self._exclude_estimators,
```

Comment:
```suggestion
            include_estimators=self._include_estimators,
```

Reply:
Thanks for the catch, I added a test for this and fixed the problem.